scripts/build-upload-docker-images.sh

```diff
@@ -2,16 +2,22 @@
 
 set -euxf -o pipefail
 
+mode=${1-main}
```

Comment:
I passed the `pr-only` arg to the script when pull-request, and set the default arg as `main` (to avoid unbound variable error).

Not sure if it is good, or there is better way to deal with this?